wallet-core-travel/src/app.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * @Description: 
+ * @Version: 1.0
+ * @Autor: z.cejay@gmail.com
+ * @Date: 2022-06-22 16:15:42
+ * @LastEditors: cejay
+ * @LastEditTime: 2022-07-26 10:52:31
+ */
+import Web3 from 'web3';
+import { Utils } from './utils/utils';
+import fs from 'fs';
+import { UserOperation } from './entity/userOperation';
+import { EntryPointAddress, Create2Factory, AddressZero } from './defines';
+import { signUserOp, getRequestId } from './utils/UserOp';
+
+async function main() {
+
+    /**
+     * web3 instance
+     */
+    const web3 = new Web3('https://ropsten.infura.io/v3/21d2f7b2f3cd4afca4c6d1930ec801c4');
+
+    // get chainId ( use for generate signature)
+    const chainId = await web3.eth.net.getId();
+    console.log(`chainId: ${chainId}`);
+
+    /**
+     * contract bytecode.contract template (test code)
+     */
+    const bytecodeTemplate = fs.readFileSync('./src/bytecodes/simpleWallet.txt', 'utf8');
+
+    // generate control key for EIP-4337 contract
+    const _account = web3.eth.accounts.create();
+    const address = _account.address;
+    const privateKey = _account.privateKey;
+    console.log(`Address: ${address}, PrivateKey: ${privateKey}`);
+
+    // generate contract wallet address
+    // create bytecode with constructor params(entry point address, control key[address])
+    const initCode = `${bytecodeTemplate}000000000000000000000000${EntryPointAddress.substring(2).toLowerCase()}000000000000000000000000${address.substring(2).toLowerCase()}`;
```

Comment:
It was a fixed method to get initCode? we can set it to Utils? or only here use this?